ground/gcs/src/plugins/setupwizard/connectiondiagram.cpp

```diff
@@ -141,6 +137,10 @@ void ConnectionDiagram::setupGraphicsScene()
             break;
         }
 
+        Core::IBoardType* type = m_configSource->getControllerType();
+        if (type != NULL)
+            elementsToShow << QString("controller-").append(type->shortName().toLower());

```

Comment:
It adds the item to the list last so it ends up with a higher z-index later, which makes it appear over top of all other elements instead of under them.